app/assets/stylesheets/navigation.scss

```diff
@@ -0,0 +1,71 @@
+$navbar-brand-font-size: 1.5rem !default;
+$side-nav-width: 320px;
+
+nav.top-nav {
+  background-color: $background-color;
+}
+
+.side-nav.fixed {
+  $bg-color: color("grey", "lighten-3");
+  $link-color: color("grey", "darken-1");
+  $line-height: 64px;
+
+  width: $side-nav-width;
+  background-color: $bg-color;
+
+  li.user-area {
+    img.avatar {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3